src/utils/date.ts

```diff
@@ -89,7 +89,10 @@ export const parseTime = (millisecond: number) => {
   return `${second.toFixed(2)} s`
 }
 
-export const parseHourFromMinute = (minutes: number | string) => {
+export const parseHourFromMinute = (minutes: number | string | undefined) => {
+  if (!minutes) {
+    return 0
+  }
```

Comment:
I think it would be better to let the caller decide what to display when the value is null, rather than having this basic utility function implement the default value.

Example:
```javascript
parseHourFromMinute(dataList[0].name ?? '0')
```